ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  golangci-lint:
```

Comment:
since you can remove golangci-lint job, we can also remove this base_image from here too.